scex-util/src/main/scala/com/avsystem/scex/util/function/NetFunctions.java

```diff
@@ -116,7 +116,7 @@ public static boolean isIps(final String ips) {
         }
 
         if (ips != null) {
-            for (String ip : ips.replaceAll(",\\s+", ",").split(",")) {
+            for (String ip : ips.trim().split("\\s*,\\s*")) {
```

Comment:
Wouldn't you like to trim each element independently?
Also, be careful with `split` - it returns an array containing empty string if input is empty.